3rdparty/libspng/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# ----------------------------------------------------------------------------
+#  CMake file for libspng. See root CMakeLists.txt
+#
+# ----------------------------------------------------------------------------
+
+project(${SPNG_LIBRARY})
+
+set(CURR_INCLUDE_DIR "${CMAKE_CURRENT_LIST_DIR}")
+set_property(GLOBAL PROPERTY SPNG_INCLUDE_DIR ${CURR_INCLUDE_DIR})
+ocv_include_directories(${CURR_INCLUDE_DIR}, ${ZLIB_INCLUDE_DIRS})
```

Comment:
> `,`

Does it work at all? There is no commas for arguments in CMake scripts.

Reply:
If it doesn't work then probably it is just not needed at all.
So `${CURR_INCLUDE_DIR}` could be dropped completely.